packages/timeline-state-resolver/src/service/remoteDeviceInstance.ts

```diff
@@ -125,11 +125,11 @@ export class RemoteDeviceInstance<
 		getCurrentTime: () => number,
 		threadConfig?: ThreadedClassConfig
 	): Promise<RemoteDeviceInstance<TOptions>> {
-		if (process.env.JEST_WORKER_ID !== undefined && threadConfig && threadConfig.disableMultithreading) {
-			// running in Jest test environment.
-			// hack: we need to work around the mangling performed by threadedClass, as getCurrentTime needs to not return a promise
-			getCurrentTime = { inner: getCurrentTime } as any
-		}
+		// if (process.env.JEST_WORKER_ID !== undefined && threadConfig && threadConfig.disableMultithreading) {
```

Comment:
If this isn't needed, perhaps teh code should be removed?

Reply:
Yeah, probably. I just don't understand why it is not necessary now when it was before.